helm/logging-operator/templates/deployment.yaml

```
@@ -41,6 +41,12 @@ spec:
           - -logging-agent={{ .Values.loggingOperator.loggingAgent }}
           - -events-logger={{ .Values.loggingOperator.eventsLogger }}
           - -default-namespaces={{ .Values.loggingOperator.defaultNamespaces }}
+          {{- if .Values.loggingOperator.excludeEventsFromNamespaces }}
+          - -exclude-events-from-namespaces={{ .Values.loggingOperator.excludeEventsFromNamespaces | join "," }}
+          {{- end }}
+          {{- if .Values.loggingOperator.includeEventsInNamespaces }}
+          - -include-events-in-namespaces={{ .Values.loggingOperator.includeEventsInNamespaces | join "," }}
```

Why is one named "IN" namespace and the other one "FROM" namespace?

```suggestion
          - -exclude-events-from-namespaces={{ .Values.loggingOperator.excludeEventsFromNamespaces | join "," }}
          {{- end }}
          {{- if .Values.loggingOperator.includeEventsInNamespaces }}
          - -include-events-in-namespaces={{ .Values.loggingOperator.includeEventsInNamespaces | join "," }}
```

---

Because I think semantically we say included in and excluded from

---

But we could say events-in-namespaces and events-not-in-namespaces

---

Oh, I was thinking we include the events coming from some namespaces (include events from namespaces) or we exclude the events coming from some namespaces.
Same semantics.